tests/python/unittest/test_numpy_op.py

```diff
@@ -1296,10 +1296,9 @@ def hybrid_forward(self, F, a, mask, value):
 
     for hybridize in [False]:
         for config in configs:
-            print(config)
             dshape, mshape, start_axis = config
             test_data = np.random.uniform(size=dshape)
-            mx_mask = np.around(np.random.uniform(size=mshape))
+            mx_mask = np.around(np.random.uniform(size=mshape)).astype(np.bool)
```

Comment:
Nitpicking:

```suggestion
            mx_mask = np.random.choice(a=[False, True], size=mshape)
```

Reply:
Thanks for your suggestions. I added this change just now : )